calculator.js

```diff
@@ -518,3 +243,38 @@ new Vue({
     },
   },
 });
+
+// This constructor returns a drink size object, optionally allowing you to
```

Comment:
Oooooh I love this. I was initially leaving it as more of a database-style data structure because I assumed that there would be one added, but this is DEFINITELY way better javascript code... 

Curious why sometimes you decided to calculate the mL and sometimes you didn't... my instinct would have been to always calculate it?

Reply:
My reasoning was that sometimes convention doesn't allow them to be consistent, e.g. a Twenty-Sixer should obviously be 26 oz, yet when you go to buy one it's generally 750 mL, which is 25.3 oz.